js/mocha-tests/package.json

```diff
@@ -0,0 +1,99 @@
+{
+  "name": "mocha-tests",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "lint": "eslint --ignore-path .gitignore .",
```

Comment:
I tend to include `eslint` in my projects by default, but this is mostly because I use VSCode which automatically fixes lint warnings/errors (same reason I use `.jsbeautifyrc` and `.editorconfig`). Since not every one uses VSCode though, I'll remove the linting from the template.